### ISSUE TITLE
Meteor Cordova build issue

### ISSUE THREAD

**Issue by ikostic** [nordnet/cordova-universal-links-plugin#52]
Hi,

i experienced problems during cordova build process. 

I received message:

```
'config.xml not found! Please, check that it exist\'s in your project\'s root directory.'
```

I debugged process and locate the problem in xmlHelper / readXmlAsJson method.

I resolved the issue with following patch:

```
xmlParser.parseString(xmlData.toString(), function(err, data) {
```

I posted this just to have it here documented for community. You can close the issue.

Thanks

**Comment by nikDemyankov** [comment]
Thanks, will fix that.

**Comment by nikDemyankov** [comment]
Fixed in v1.1.1. Version is released.

**Comment by rjakobsson** [comment]
Outside of the scope of this issue, @ikostic how did you add your configs in mobile-config.js?
